src/popup/components/TransactionAndTokenFilter.vue

```diff
@@ -95,7 +100,7 @@ export default defineComponent({
 
     onMounted(() => {
       if (showFilterBar.value) {
-        maxHeight.value = (refs.transactionFilter as HTMLDivElement).clientHeight;
+        maxHeight.value = (instance?.refs?.transactionFilter as HTMLDivElement)?.clientHeight;
```

Comment:
Please use Vue 3 approach to create DOM element refs: https://vuejs.org/guide/essentials/template-refs.html#accessing-the-refs